pages/add-sponsor.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { AddSponsor } from '../client/components';
+
+const AddSponsorPage = () => {
+  // TODO - Dynamically get Chapter and EventId s
+  const eventId = '123abc';
+  const chapterId = 'abc123';
+
+  return (
+    <div>
+      <AddSponsor eventId={eventId} chapterId={chapterId} />
+      <style jsx>{`
```

Comment:
I believe we switched from using styled jsx to styled components

Reply:
yes we did (praise!) however I just copy/pasted the existing index page :)